[2023-12-03T18:58:24.406+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T18:58:24.407+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T18:58:24.410+0000] {logging_mixin.py:154} INFO - [2023-12-03T18:58:24.409+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T18:58:24.483+0000] {logging_mixin.py:154} INFO - [2023-12-03T18:58:24.480+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T18:58:24.544+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T18:58:32.988+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 8.588 seconds
[2023-12-03T18:59:03.375+0000] {processor.py:157} INFO - Started process (PID=96) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T18:59:03.376+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T18:59:03.380+0000] {logging_mixin.py:154} INFO - [2023-12-03T18:59:03.380+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T18:59:03.409+0000] {logging_mixin.py:154} INFO - [2023-12-03T18:59:03.404+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T18:59:03.410+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T18:59:03.429+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.059 seconds
[2023-12-03T18:59:33.663+0000] {processor.py:157} INFO - Started process (PID=151) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T18:59:33.664+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T18:59:33.667+0000] {logging_mixin.py:154} INFO - [2023-12-03T18:59:33.667+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T18:59:33.687+0000] {logging_mixin.py:154} INFO - [2023-12-03T18:59:33.683+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T18:59:33.688+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T18:59:33.707+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.049 seconds
[2023-12-03T19:00:03.889+0000] {processor.py:157} INFO - Started process (PID=206) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:00:03.890+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:00:03.892+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:00:03.892+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:00:03.910+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:00:03.907+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T19:00:03.912+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:00:03.932+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.046 seconds
[2023-12-03T19:00:34.445+0000] {processor.py:157} INFO - Started process (PID=261) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:00:34.446+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:00:34.448+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:00:34.447+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:00:34.464+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:00:34.461+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T19:00:34.465+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:00:34.479+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.036 seconds
[2023-12-03T19:01:04.899+0000] {processor.py:157} INFO - Started process (PID=316) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:01:04.901+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:01:04.904+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:01:04.904+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:01:04.941+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:01:04.935+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T19:01:04.943+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:01:04.981+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.088 seconds
[2023-12-03T19:01:35.300+0000] {processor.py:157} INFO - Started process (PID=371) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:01:35.301+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:01:35.304+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:01:35.304+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:01:35.335+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:01:35.328+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T19:01:35.338+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:01:35.362+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.065 seconds
[2023-12-03T19:02:05.933+0000] {processor.py:157} INFO - Started process (PID=426) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:02:05.934+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:02:05.936+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:02:05.936+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:02:05.951+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:02:05.948+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T19:02:05.952+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:02:05.973+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.043 seconds
[2023-12-03T19:02:36.091+0000] {processor.py:157} INFO - Started process (PID=481) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:02:36.092+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:02:36.094+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:02:36.094+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:02:36.113+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:02:36.109+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T19:02:36.115+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:02:36.139+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.051 seconds
[2023-12-03T19:03:06.444+0000] {processor.py:157} INFO - Started process (PID=537) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:03:06.445+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:03:06.447+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:03:06.447+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:03:06.463+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:03:06.460+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T19:03:06.464+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:03:06.478+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.036 seconds
[2023-12-03T19:03:36.704+0000] {processor.py:157} INFO - Started process (PID=592) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:03:36.705+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:03:36.707+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:03:36.707+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:03:36.725+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:03:36.721+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T19:03:36.726+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:03:36.740+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.039 seconds
[2023-12-03T19:04:06.951+0000] {processor.py:157} INFO - Started process (PID=646) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:04:06.951+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:04:06.953+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:04:06.953+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:04:06.969+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:04:06.966+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=api_call,
UnboundLocalError: local variable 'api_call' referenced before assignment
[2023-12-03T19:04:06.970+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:04:06.988+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.040 seconds
[2023-12-03T19:04:12.929+0000] {processor.py:157} INFO - Started process (PID=657) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:04:12.930+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:04:12.932+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:04:12.931+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:04:12.959+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:04:12.956+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=_api_call,
UnboundLocalError: local variable '_api_call' referenced before assignment
[2023-12-03T19:04:12.961+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:04:12.976+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.050 seconds
[2023-12-03T19:04:43.213+0000] {processor.py:157} INFO - Started process (PID=712) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:04:43.214+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:04:43.216+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:04:43.215+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:04:43.234+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:04:43.230+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=_api_call,
UnboundLocalError: local variable '_api_call' referenced before assignment
[2023-12-03T19:04:43.235+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:04:43.259+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.049 seconds
[2023-12-03T19:05:13.295+0000] {processor.py:157} INFO - Started process (PID=767) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:05:13.296+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:05:13.297+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:05:13.297+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:05:13.313+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:05:13.310+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=_api_call,
UnboundLocalError: local variable '_api_call' referenced before assignment
[2023-12-03T19:05:13.314+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:05:13.334+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.041 seconds
[2023-12-03T19:05:43.961+0000] {processor.py:157} INFO - Started process (PID=829) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:05:43.962+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:05:43.964+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:05:43.963+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:05:44.092+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:05:44.088+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 40, in city_group
    python_callable=_api_call,
UnboundLocalError: local variable '_api_call' referenced before assignment
[2023-12-03T19:05:44.093+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:05:44.107+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.148 seconds
[2023-12-03T19:06:06.213+0000] {processor.py:157} INFO - Started process (PID=867) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:06:06.214+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:06:06.217+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:06:06.216+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:06:06.254+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:06:06.250+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 52, in city_group
    "appid": Variable.get("WEATHER_API_KEY"),
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/variable.py", line 140, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable WEATHER_API_KEY does not exist'
[2023-12-03T19:06:06.255+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:06:06.264+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.053 seconds
[2023-12-03T19:06:36.730+0000] {processor.py:157} INFO - Started process (PID=922) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:06:36.733+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:06:36.735+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:06:36.735+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:06:36.804+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:06:36.791+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 52, in city_group
    "appid": Variable.get("WEATHER_API_KEY"),
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/variable.py", line 140, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable WEATHER_API_KEY does not exist'
[2023-12-03T19:06:36.805+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:06:36.818+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.091 seconds
[2023-12-03T19:07:06.942+0000] {processor.py:157} INFO - Started process (PID=975) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:07:06.944+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:07:06.949+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:07:06.948+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:07:06.985+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:07:06.982+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 52, in city_group
    "appid": Variable.get("WEATHER_API_KEY"),
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/variable.py", line 140, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable WEATHER_API_KEY does not exist'
[2023-12-03T19:07:06.987+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:07:06.996+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.058 seconds
[2023-12-03T19:07:25.082+0000] {processor.py:157} INFO - Started process (PID=1017) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:07:25.084+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:07:25.088+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:07:25.088+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:07:25.144+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:07:25.136+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 59, in city_group
    _api_call >> extract_data
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 108, in __rrshift__
    self.__lshift__(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 98, in __lshift__
    self.set_upstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 269, in set_upstream
    self._set_relatives(task_or_task_list, upstream=True, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 213, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2023-12-03T19:07:25.146+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:07:25.181+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.107 seconds
[2023-12-03T19:07:55.268+0000] {processor.py:157} INFO - Started process (PID=1072) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:07:55.269+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:07:55.271+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:07:55.271+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:07:55.288+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:07:55.285+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 59, in city_group
    _api_call >> extract_data
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 108, in __rrshift__
    self.__lshift__(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 98, in __lshift__
    self.set_upstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 269, in set_upstream
    self._set_relatives(task_or_task_list, upstream=True, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 213, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2023-12-03T19:07:55.288+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:07:55.306+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.041 seconds
[2023-12-03T19:08:26.014+0000] {processor.py:157} INFO - Started process (PID=1127) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:08:26.015+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:08:26.018+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:08:26.017+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:08:26.036+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:08:26.032+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 59, in city_group
    _api_call >> extract_data
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 108, in __rrshift__
    self.__lshift__(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 98, in __lshift__
    self.set_upstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 269, in set_upstream
    self._set_relatives(task_or_task_list, upstream=True, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 213, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2023-12-03T19:08:26.036+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:08:26.056+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.046 seconds
[2023-12-03T19:08:56.205+0000] {processor.py:157} INFO - Started process (PID=1182) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:08:56.206+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:08:56.210+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:08:56.209+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:08:56.236+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:08:56.232+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 59, in city_group
    _api_call >> extract_data
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 108, in __rrshift__
    self.__lshift__(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 98, in __lshift__
    self.set_upstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 269, in set_upstream
    self._set_relatives(task_or_task_list, upstream=True, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 213, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2023-12-03T19:08:56.237+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:08:56.259+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.059 seconds
[2023-12-03T19:09:26.360+0000] {processor.py:157} INFO - Started process (PID=1237) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:09:26.363+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:09:26.367+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:09:26.366+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:09:26.401+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:09:26.396+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <module>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 120, in <listcomp>
    extract_groups = [create_group(city)() for city in coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 59, in city_group
    _api_call >> extract_data
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 108, in __rrshift__
    self.__lshift__(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 98, in __lshift__
    self.set_upstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 269, in set_upstream
    self._set_relatives(task_or_task_list, upstream=True, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 213, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2023-12-03T19:09:26.402+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:09:26.435+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.081 seconds
[2023-12-03T19:09:37.645+0000] {processor.py:157} INFO - Started process (PID=1251) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:09:37.646+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:09:37.650+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:09:37.649+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:09:37.688+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:09:37.685+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 42, in city_group
    "appid": Variable.get("WEATHER_API_KEY"),
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/variable.py", line 140, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable WEATHER_API_KEY does not exist'
[2023-12-03T19:09:37.689+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:09:37.697+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.055 seconds
[2023-12-03T19:10:07.871+0000] {processor.py:157} INFO - Started process (PID=1306) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:10:07.871+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:10:07.873+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:07.873+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:10:07.901+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:07.898+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 42, in city_group
    "appid": Variable.get("WEATHER_API_KEY"),
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/variable.py", line 140, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable WEATHER_API_KEY does not exist'
[2023-12-03T19:10:07.902+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:10:07.915+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.047 seconds
[2023-12-03T19:10:16.933+0000] {processor.py:157} INFO - Started process (PID=1326) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:10:16.934+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:10:16.937+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:16.936+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:10:16.972+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:10:17.138+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:17.138+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-12-03T19:10:17.157+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:17.157+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-12-03T19:10:17.175+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:17.175+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-12-03T19:10:17.176+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:17.176+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:10:17.189+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:17.188+0000] {dag.py:2963} INFO - Creating ORM DAG for weather_scrapper
[2023-12-03T19:10:17.199+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:17.199+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-11-23T00:00:00+00:00, run_after=2023-11-24T00:00:00+00:00
[2023-12-03T19:10:17.220+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.289 seconds
[2023-12-03T19:10:47.530+0000] {processor.py:157} INFO - Started process (PID=1385) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:10:47.532+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:10:47.537+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:47.536+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:10:47.589+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:10:47.635+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:47.635+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:10:47.677+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:10:47.677+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:10:47.717+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.195 seconds
[2023-12-03T19:11:17.801+0000] {processor.py:157} INFO - Started process (PID=1440) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:11:17.802+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:11:17.804+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:11:17.804+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:11:17.827+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:11:17.850+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:11:17.850+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:11:17.870+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:11:17.870+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:11:17.890+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.094 seconds
[2023-12-03T19:11:48.508+0000] {processor.py:157} INFO - Started process (PID=1495) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:11:48.509+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:11:48.511+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:11:48.511+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:11:48.545+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:11:48.572+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:11:48.571+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:11:48.593+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:11:48.593+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:11:48.613+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.107 seconds
[2023-12-03T19:12:19.008+0000] {processor.py:157} INFO - Started process (PID=1550) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:12:19.009+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:12:19.011+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:12:19.011+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:12:19.036+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:12:19.061+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:12:19.060+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:12:19.086+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:12:19.085+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:12:19.111+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.106 seconds
[2023-12-03T19:12:49.434+0000] {processor.py:157} INFO - Started process (PID=1605) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:12:49.435+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:12:49.438+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:12:49.437+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:12:49.463+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:12:49.489+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:12:49.489+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:12:49.514+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:12:49.514+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:12:49.539+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.108 seconds
[2023-12-03T19:13:19.706+0000] {processor.py:157} INFO - Started process (PID=1660) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:13:19.707+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:13:19.709+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:13:19.709+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:13:19.731+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:13:19.753+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:13:19.753+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:13:19.775+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:13:19.775+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:13:19.794+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.092 seconds
[2023-12-03T19:13:49.840+0000] {processor.py:157} INFO - Started process (PID=1715) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:13:49.841+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:13:49.843+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:13:49.843+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:13:49.866+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:13:49.890+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:13:49.889+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:13:49.910+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:13:49.910+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:13:49.929+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.092 seconds
[2023-12-03T19:14:20.280+0000] {processor.py:157} INFO - Started process (PID=1770) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:14:20.284+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:14:20.290+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:14:20.289+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:14:20.325+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:14:20.363+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:14:20.363+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:14:20.412+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:14:20.412+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:14:20.446+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.174 seconds
[2023-12-03T19:14:50.764+0000] {processor.py:157} INFO - Started process (PID=1825) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:14:50.764+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:14:50.766+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:14:50.766+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:14:50.790+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:14:50.814+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:14:50.814+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:14:50.845+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:14:50.845+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:14:50.865+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.105 seconds
[2023-12-03T19:15:20.936+0000] {processor.py:157} INFO - Started process (PID=1879) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:15:20.938+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:15:20.940+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:15:20.940+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:15:20.974+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:15:21.002+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:15:21.002+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:15:21.047+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:15:21.046+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:15:21.070+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.136 seconds
[2023-12-03T19:15:51.681+0000] {processor.py:157} INFO - Started process (PID=1931) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:15:51.682+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:15:51.685+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:15:51.684+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:15:51.715+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:15:51.740+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:15:51.739+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:15:51.764+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:15:51.764+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:15:51.783+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.105 seconds
[2023-12-03T19:16:21.945+0000] {processor.py:157} INFO - Started process (PID=1987) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:16:21.946+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:16:21.948+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:16:21.948+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:16:21.976+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:16:22.013+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:16:22.012+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:16:22.036+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:16:22.036+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:16:22.054+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.113 seconds
[2023-12-03T19:16:52.849+0000] {processor.py:157} INFO - Started process (PID=2042) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:16:52.850+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:16:52.852+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:16:52.852+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:16:52.877+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:16:52.906+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:16:52.906+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:16:52.930+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:16:52.929+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:16:52.947+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.101 seconds
[2023-12-03T19:17:23.039+0000] {processor.py:157} INFO - Started process (PID=2097) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:17:23.040+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:17:23.042+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:17:23.042+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:17:23.065+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:17:23.087+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:17:23.086+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:17:23.108+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:17:23.108+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:17:23.125+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.090 seconds
[2023-12-03T19:17:53.749+0000] {processor.py:157} INFO - Started process (PID=2152) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:17:53.750+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:17:53.752+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:17:53.751+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:17:53.774+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:17:53.799+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:17:53.799+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:17:53.820+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:17:53.820+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:17:53.843+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.097 seconds
[2023-12-03T19:18:24.028+0000] {processor.py:157} INFO - Started process (PID=2207) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:18:24.029+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:18:24.031+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:18:24.031+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:18:24.060+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:18:24.088+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:18:24.088+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:18:24.114+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:18:24.114+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:18:24.135+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.111 seconds
[2023-12-03T19:18:54.328+0000] {processor.py:157} INFO - Started process (PID=2262) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:18:54.330+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:18:54.335+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:18:54.334+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:18:54.378+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:18:54.417+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:18:54.417+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:18:54.454+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:18:54.453+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:18:54.486+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.166 seconds
[2023-12-03T19:19:25.426+0000] {processor.py:157} INFO - Started process (PID=2317) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:19:25.427+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:19:25.429+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:19:25.429+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:19:25.452+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:19:25.473+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:19:25.473+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:19:25.495+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:19:25.495+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:19:25.513+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.091 seconds
[2023-12-03T19:19:56.232+0000] {processor.py:157} INFO - Started process (PID=2372) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:19:56.233+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:19:56.235+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:19:56.235+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:19:56.260+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:19:56.292+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:19:56.292+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:19:56.318+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:19:56.317+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:19:56.345+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.116 seconds
[2023-12-03T19:20:26.881+0000] {processor.py:157} INFO - Started process (PID=2427) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:20:26.881+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:20:26.884+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:20:26.883+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:20:26.912+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:20:26.941+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:20:26.941+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:20:26.986+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:20:26.985+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:20:27.010+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.132 seconds
[2023-12-03T19:20:57.111+0000] {processor.py:157} INFO - Started process (PID=2482) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:20:57.113+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:20:57.117+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:20:57.117+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:20:57.158+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:20:57.189+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:20:57.189+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:20:57.235+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:20:57.235+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:20:57.269+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.165 seconds
[2023-12-03T19:21:27.498+0000] {processor.py:157} INFO - Started process (PID=2537) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:21:27.499+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:21:27.502+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:21:27.501+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:21:27.525+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:21:27.547+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:21:27.547+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:21:27.568+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:21:27.567+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:21:27.590+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.095 seconds
[2023-12-03T19:21:58.349+0000] {processor.py:157} INFO - Started process (PID=2592) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:21:58.351+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:21:58.353+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:21:58.352+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:21:58.377+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:21:58.399+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:21:58.399+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:21:58.421+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:21:58.421+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:21:58.441+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.096 seconds
[2023-12-03T19:22:28.792+0000] {processor.py:157} INFO - Started process (PID=2647) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:22:28.793+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:22:28.796+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:22:28.795+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:22:28.821+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:22:28.845+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:22:28.845+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:22:28.867+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:22:28.867+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:22:28.892+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.103 seconds
[2023-12-03T19:22:59.480+0000] {processor.py:157} INFO - Started process (PID=2702) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:22:59.481+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:22:59.483+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:22:59.483+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:22:59.508+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:22:59.532+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:22:59.532+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:22:59.558+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:22:59.557+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:22:59.581+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.104 seconds
[2023-12-03T19:23:29.877+0000] {processor.py:157} INFO - Started process (PID=2757) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:23:29.878+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:23:29.880+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:23:29.880+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:23:29.907+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:23:29.930+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:23:29.930+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:23:29.952+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:23:29.952+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:23:29.970+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.096 seconds
[2023-12-03T19:24:00.567+0000] {processor.py:157} INFO - Started process (PID=2812) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:24:00.568+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:24:00.570+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:24:00.570+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:24:00.594+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:24:00.617+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:24:00.617+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:24:00.638+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:24:00.637+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:24:00.662+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.098 seconds
[2023-12-03T19:24:30.708+0000] {processor.py:157} INFO - Started process (PID=2867) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:24:30.709+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:24:30.712+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:24:30.711+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:24:30.738+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:24:30.762+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:24:30.762+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:24:30.786+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:24:30.786+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:24:30.806+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.102 seconds
[2023-12-03T19:25:01.432+0000] {processor.py:157} INFO - Started process (PID=2922) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:25:01.433+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:25:01.435+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:25:01.435+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:25:01.464+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:25:01.487+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:25:01.487+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:25:01.509+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:25:01.509+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:25:01.529+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.100 seconds
[2023-12-03T19:25:31.864+0000] {processor.py:157} INFO - Started process (PID=2977) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:25:31.865+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:25:31.868+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:25:31.867+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:25:31.896+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:25:31.924+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:25:31.924+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:25:31.949+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:25:31.949+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:25:31.967+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.108 seconds
[2023-12-03T19:26:02.234+0000] {processor.py:157} INFO - Started process (PID=3033) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:26:02.235+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:26:02.237+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:26:02.237+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:26:02.263+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:26:02.286+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:26:02.286+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:26:02.306+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:26:02.306+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:26:02.332+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.102 seconds
[2023-12-03T19:26:32.749+0000] {processor.py:157} INFO - Started process (PID=3089) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:26:32.751+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:26:32.754+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:26:32.754+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:26:32.779+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:26:32.805+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:26:32.805+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:26:32.834+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:26:32.834+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:26:32.850+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.106 seconds
[2023-12-03T19:27:03.015+0000] {processor.py:157} INFO - Started process (PID=3158) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:27:03.016+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:27:03.019+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:27:03.019+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:27:03.042+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:27:03.072+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:27:03.072+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:27:03.095+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:27:03.095+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:27:03.124+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.111 seconds
[2023-12-03T19:27:33.357+0000] {processor.py:157} INFO - Started process (PID=3213) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:27:33.357+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:27:33.359+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:27:33.359+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:27:33.382+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:27:33.403+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:27:33.403+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:27:33.425+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:27:33.425+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:27:33.449+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.096 seconds
[2023-12-03T19:28:04.284+0000] {processor.py:157} INFO - Started process (PID=3268) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:28:04.285+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:28:04.287+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:28:04.286+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:28:04.319+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:28:04.359+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:28:04.358+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:28:04.383+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:28:04.383+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:28:04.409+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.128 seconds
[2023-12-03T19:28:34.716+0000] {processor.py:157} INFO - Started process (PID=3323) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:28:34.716+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:28:34.720+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:28:34.719+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:28:34.757+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:28:34.791+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:28:34.791+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:28:34.817+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:28:34.817+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:28:34.852+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.139 seconds
[2023-12-03T19:29:05.024+0000] {processor.py:157} INFO - Started process (PID=3378) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:29:05.026+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:29:05.031+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:29:05.030+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:29:05.080+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:29:05.196+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:29:05.196+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:29:05.319+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:29:05.318+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:29:05.361+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.342 seconds
[2023-12-03T19:29:35.496+0000] {processor.py:157} INFO - Started process (PID=3434) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:29:35.497+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:29:35.500+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:29:35.499+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:29:35.529+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:29:35.554+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:29:35.554+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:29:35.578+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:29:35.578+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:29:35.599+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.106 seconds
[2023-12-03T19:30:06.150+0000] {processor.py:157} INFO - Started process (PID=3495) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:30:06.154+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:30:06.159+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:30:06.158+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:30:06.250+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:30:06.313+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:30:06.313+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:30:06.367+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:30:06.366+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:30:06.416+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.278 seconds
[2023-12-03T19:30:36.515+0000] {processor.py:157} INFO - Started process (PID=3571) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:30:36.517+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:30:36.519+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:30:36.519+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:30:36.548+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:30:36.578+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:30:36.578+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:30:36.604+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:30:36.604+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:30:36.634+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.123 seconds
[2023-12-03T19:31:06.862+0000] {processor.py:157} INFO - Started process (PID=3626) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:31:06.864+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:31:06.866+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:31:06.866+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:31:06.896+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:31:06.930+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:31:06.930+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:31:06.960+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:31:06.960+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:31:06.986+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.129 seconds
[2023-12-03T19:31:37.796+0000] {processor.py:157} INFO - Started process (PID=3681) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:31:37.797+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:31:37.799+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:31:37.799+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:31:37.832+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:31:37.860+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:31:37.860+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:31:37.886+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:31:37.886+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:31:37.908+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.115 seconds
[2023-12-03T19:32:08.188+0000] {processor.py:157} INFO - Started process (PID=3736) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:32:08.190+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:32:08.192+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:32:08.192+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:32:08.219+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:32:08.252+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:32:08.252+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:32:08.280+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:32:08.280+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:32:08.306+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.121 seconds
[2023-12-03T19:32:38.756+0000] {processor.py:157} INFO - Started process (PID=3791) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:32:38.758+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:32:38.763+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:32:38.762+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:32:38.798+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:32:38.823+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:32:38.822+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:32:38.846+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:32:38.845+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:32:38.871+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.120 seconds
[2023-12-03T19:33:09.521+0000] {processor.py:157} INFO - Started process (PID=3846) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:09.522+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:33:09.526+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:09.525+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:09.560+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:09.600+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:09.600+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:33:09.624+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:09.623+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:33:09.652+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.135 seconds
[2023-12-03T19:33:15.743+0000] {processor.py:157} INFO - Started process (PID=3867) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:15.745+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:33:15.750+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:15.749+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:15.793+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:15.789+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 89, in __call__
    return self._create_task_group(TaskGroup, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/task_group.py", line 97, in _create_task_group
    retval = self.function(*args, **kwargs)
  File "/opt/airflow/dags/weather_scrapper.py", line 49, in city_group
    prepare_api_call >> extract_data
NameError: name 'prepare_api_call' is not defined
[2023-12-03T19:33:15.794+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:15.813+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.072 seconds
[2023-12-03T19:33:32.062+0000] {processor.py:157} INFO - Started process (PID=3897) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:32.063+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:33:32.065+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:32.065+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:32.105+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:32.220+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:32.220+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:33:32.240+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:32.240+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:33:32.270+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.210 seconds
[2023-12-03T19:33:50.091+0000] {processor.py:157} INFO - Started process (PID=3937) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:50.092+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:33:50.094+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:50.094+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:50.122+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:33:50.119+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:33:50.123+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:33:50.139+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.051 seconds
[2023-12-03T19:34:20.269+0000] {processor.py:157} INFO - Started process (PID=3988) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:34:20.270+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:34:20.272+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:34:20.272+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:34:20.291+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:34:20.288+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:34:20.292+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:34:20.316+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.050 seconds
[2023-12-03T19:34:40.562+0000] {processor.py:157} INFO - Started process (PID=4010) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:34:40.564+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:34:40.567+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:34:40.567+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:34:40.602+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:34:40.599+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:34:40.604+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:34:40.628+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.070 seconds
[2023-12-03T19:35:11.069+0000] {processor.py:157} INFO - Started process (PID=4065) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:35:11.070+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:35:11.073+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:35:11.073+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:35:11.097+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:35:11.093+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in city_coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:35:11.099+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:35:11.124+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.058 seconds
[2023-12-03T19:35:35.402+0000] {processor.py:157} INFO - Started process (PID=4118) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:35:35.406+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:35:35.409+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:35:35.409+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:35:35.441+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:35:35.437+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:35:35.442+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:35:35.464+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.064 seconds
[2023-12-03T19:36:05.874+0000] {processor.py:157} INFO - Started process (PID=4173) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:36:05.875+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:36:05.876+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:36:05.876+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:36:05.902+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:36:05.897+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:36:05.904+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:36:05.920+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.048 seconds
[2023-12-03T19:36:36.481+0000] {processor.py:157} INFO - Started process (PID=4228) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:36:36.482+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:36:36.483+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:36:36.483+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:36:36.505+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:36:36.501+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:36:36.506+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:36:36.524+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.046 seconds
[2023-12-03T19:36:53.351+0000] {processor.py:157} INFO - Started process (PID=4267) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:36:53.352+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:36:53.353+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:36:53.353+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:36:53.394+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:36:53.391+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:36:53.395+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:36:53.412+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.064 seconds
[2023-12-03T19:37:07.197+0000] {processor.py:157} INFO - Started process (PID=4284) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:37:07.198+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:37:07.200+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:37:07.199+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:37:07.237+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:37:07.233+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:37:07.239+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:37:07.258+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.066 seconds
[2023-12-03T19:37:37.680+0000] {processor.py:157} INFO - Started process (PID=4339) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:37:37.681+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:37:37.682+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:37:37.682+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:37:37.706+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:37:37.702+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:37:37.708+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:37:37.725+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.048 seconds
[2023-12-03T19:37:43.803+0000] {processor.py:157} INFO - Started process (PID=4342) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:37:43.805+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:37:43.806+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:37:43.806+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:37:43.841+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:37:43.837+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:37:43.843+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:37:43.858+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.059 seconds
[2023-12-03T19:38:04.430+0000] {processor.py:157} INFO - Started process (PID=4394) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:38:04.432+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:38:04.433+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:38:04.433+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:38:04.481+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:38:04.475+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:38:04.483+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:38:04.503+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.076 seconds
[2023-12-03T19:38:34.688+0000] {processor.py:157} INFO - Started process (PID=4449) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:38:34.689+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:38:34.690+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:38:34.690+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:38:34.709+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:38:34.705+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:38:34.710+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:38:34.725+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.040 seconds
[2023-12-03T19:38:53.447+0000] {processor.py:157} INFO - Started process (PID=4484) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:38:53.448+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:38:53.449+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:38:53.448+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:38:53.481+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:38:53.477+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 122, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:38:53.483+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:38:53.509+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.066 seconds
[2023-12-03T19:39:17.963+0000] {processor.py:157} INFO - Started process (PID=4526) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:39:17.964+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:39:17.965+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:39:17.965+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:39:17.997+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:39:17.992+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 123, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 123, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:39:17.999+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:39:18.024+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.064 seconds
[2023-12-03T19:39:48.901+0000] {processor.py:157} INFO - Started process (PID=4581) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:39:48.902+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:39:48.904+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:39:48.903+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:39:48.928+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:39:48.924+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/weather_scrapper.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/weather_scrapper.py", line 123, in <module>
    extract_groups = [create_city_groups(city)() for city in coordinates]
  File "/opt/airflow/dags/weather_scrapper.py", line 123, in <listcomp>
    extract_groups = [create_city_groups(city)() for city in coordinates]
NameError: name 'create_city_groups' is not defined
[2023-12-03T19:39:48.930+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:39:48.955+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.057 seconds
[2023-12-03T19:39:55.766+0000] {processor.py:157} INFO - Started process (PID=4596) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:39:55.767+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:39:55.768+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:39:55.768+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:39:55.805+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:39:55.908+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:39:55.907+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:39:55.934+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:39:55.934+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:39:55.974+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.210 seconds
[2023-12-03T19:40:26.700+0000] {processor.py:157} INFO - Started process (PID=4678) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:40:26.701+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:40:26.702+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:40:26.702+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:40:26.729+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:40:26.765+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:40:26.765+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:40:26.794+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:40:26.794+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:40:26.815+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.119 seconds
[2023-12-03T19:40:56.893+0000] {processor.py:157} INFO - Started process (PID=4733) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:40:56.894+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:40:56.895+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:40:56.895+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:40:56.932+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:40:56.957+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:40:56.956+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:40:56.981+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:40:56.981+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:40:57.005+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.115 seconds
[2023-12-03T19:41:06.068+0000] {processor.py:157} INFO - Started process (PID=4748) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:41:06.070+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:41:06.072+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:41:06.072+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:41:06.138+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:41:06.271+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:41:06.271+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:41:06.300+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:41:06.299+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:41:06.340+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.277 seconds
[2023-12-03T19:41:36.906+0000] {processor.py:157} INFO - Started process (PID=4834) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:41:36.908+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:41:36.909+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:41:36.909+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:41:36.946+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:41:36.971+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:41:36.971+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:41:36.998+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:41:36.997+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:41:37.026+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.123 seconds
[2023-12-03T19:42:07.156+0000] {processor.py:157} INFO - Started process (PID=4889) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:42:07.157+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:42:07.158+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:42:07.158+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:42:07.212+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:42:07.256+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:42:07.255+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:42:07.283+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:42:07.283+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:42:07.302+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.150 seconds
[2023-12-03T19:42:37.383+0000] {processor.py:157} INFO - Started process (PID=4944) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:42:37.384+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:42:37.385+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:42:37.384+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:42:37.425+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:42:37.460+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:42:37.460+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:42:37.487+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:42:37.486+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:42:37.509+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.129 seconds
[2023-12-03T19:43:08.072+0000] {processor.py:157} INFO - Started process (PID=4997) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:43:08.073+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:43:08.074+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:43:08.074+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:43:08.104+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:43:08.132+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:43:08.131+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:43:08.165+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:43:08.164+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:43:08.184+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.115 seconds
[2023-12-03T19:43:38.895+0000] {processor.py:157} INFO - Started process (PID=5052) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:43:38.896+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:43:38.897+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:43:38.897+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:43:38.935+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:43:38.969+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:43:38.968+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:43:38.996+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:43:38.995+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:43:39.021+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.133 seconds
[2023-12-03T19:44:09.341+0000] {processor.py:157} INFO - Started process (PID=5107) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:44:09.342+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:44:09.343+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:44:09.342+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:44:09.381+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:44:09.416+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:44:09.416+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:44:09.440+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:44:09.440+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:44:09.461+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.123 seconds
[2023-12-03T19:44:39.695+0000] {processor.py:157} INFO - Started process (PID=5162) to work on /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:44:39.696+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/weather_scrapper.py for tasks to queue
[2023-12-03T19:44:39.697+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:44:39.697+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:44:39.724+0000] {processor.py:839} INFO - DAG(s) dict_keys(['weather_scrapper']) retrieved from /opt/airflow/dags/weather_scrapper.py
[2023-12-03T19:44:39.756+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:44:39.756+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-03T19:44:39.795+0000] {logging_mixin.py:154} INFO - [2023-12-03T19:44:39.795+0000] {dag.py:3722} INFO - Setting next_dagrun for weather_scrapper to 2023-12-03T00:00:00+00:00, run_after=2023-12-04T00:00:00+00:00
[2023-12-03T19:44:39.819+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/weather_scrapper.py took 0.127 seconds
